### ISSUE TITLE
Ending cutscenes skip frames

### ISSUE THREAD

**Issue by renaudbedard** [renaudbedard/fez-1.12-issues#152]
The ending cutscenes use the 17ms update timesteps path and so do not look their best. This is not an easy problem to fix, but I at least want to acknowledge it because it's certainly not ideal.

**Comment by Nicholas-Steel** [comment]
Yes, I mentioned this in one of the other bug report topics I made here. It isn't as smooth as it should be and I feel that the ending cutscenes are noteworthy enough to deserve an improvement in the frame pacing to smooth it out.

**Comment by Nicholas-Steel** [comment]
This also seems to affect the "Press A" on the FEZ logo at the start of the game and also the Fez, Shades and 3D Glasses falling down to your character at the start of the game.
